gradlew

```diff
@@ -87,9 +121,9 @@ CLASSPATH=$APP_HOME/gradle/wrapper/gradle-wrapper.jar
 if [ -n "$JAVA_HOME" ] ; then
     if [ -x "$JAVA_HOME/jre/sh/java" ] ; then
         # IBM's JDK on AIX uses strange locations for the executables
-        JAVACMD="$JAVA_HOME/jre/sh/java"
+        JAVACMD=$JAVA_HOME/jre/sh/java
     else
-        JAVACMD="$JAVA_HOME/bin/java"
+        JAVACMD=$JAVA_HOME/bin/java
```

Comment:
These are autogenerated? I wonder why the quotes have been dropped- this is not safe given what we know about `$JAVA_HOME` at this point, is it? 

Going through [gradle PR #17666](https://github.com/gradle/gradle/pull/17666) it looks like there's a belief that these quotes are redundant. I don't know why though.

Reply:
Turns out field/word splitting does not occur in variable assignment context. TIL.